config/config.toml.tpl

```diff
@@ -409,8 +409,9 @@ trust_height = {{ .StateSync.TrustHeight }}
 trust_hash = "{{ .StateSync.TrustHash }}"
 trust_period = "{{ .StateSync.TrustPeriod }}"
 
-# Time to spend discovering snapshots before initiating a restore.
-discovery_time = "{{ .StateSync.DiscoveryTime }}"
+# Time to spend discovering snapshots before switching to blocksync. If set to
```

Comment:
Should we indicate here what the default is as 2min is not a short amount of time so people might want to lower this.

Reply:
We can change the default to 30 sec or 1 min. Why would people want to go lower?

Reply:
I dont think we need to change it, but I understood that the initial complaint was that they get stuck too long and want to move on.. So we leave 2 mins but just tell them. No need to change it ourselves.

Reply:
> I understood that the initial complaint was that they get stuck too long and want to move on..

yeah, because we **never** switched to blocksync before. Not after 2min, never.